pkg/sbproxy/notifications/producer.go

```diff
@@ -276,7 +273,7 @@ func (p *Producer) ping(ctx context.Context, done chan<- struct{}) {
 	}
 }
 
-func (p *Producer) connect(ctx context.Context) error {
+func (p *Producer) connect(ctx context.Context, resyncChan chan struct{}) error {
```

Comment:
why mix ws connect with resync?

Reply:
I wanted to remove the shouldResync variable but for now this is reverted